main/daal-fry/recipe.txt

```diff
@@ -0,0 +1,6 @@
+1. Wash and soak lentils.
+2. Pressure cook lentils with turmeric and salt.
+3. In a pan, heat oil and add cumin seeds, garlic, and ginger.
+4. Sauté onions until golden brown.
+5. Add tomatoes and spices, and cook until oil separates.
+6. Add cooked lentils and water, and simmer until desired consistency.
```

Comment:
Add the last step of garnishing.

Reply:
The last step of garnishing has been added.